examples/mugen/retrieval/video_clip.py

```diff
@@ -72,14 +81,115 @@ class VideoEncoder(nn.Module):
     """Encode videos to a fixed size vector. Adapted from VideoCLIP
```

Comment:
"Fixed size vector" can be ambiguous. Can we elaborate on what `VideoEncoder` does here?

Reply:
"Encode videos to the last layer before the fully-connected layer of S3D"?